### ISSUE TITLE
java.lang.NoClassDefFoundError: javafx/util/Pair

### ISSUE THREAD

**Issue by JHeimbach** [adelf/idea-php-dotenv-plugin#12]
Hey,

first of all thanks for your plugin, it is really great.

Since today my PHPstorm throws following Exception:

```
javafx/util/Pair
java.lang.NoClassDefFoundError: javafx/util/Pair
	at ru.adelf.idea.dotenv.util.EnvironmentVariablesUtil.getKeyValueFromString(EnvironmentVariablesUtil.java:21)
	at ru.adelf.idea.dotenv.DotEnvPsiElementsVisitor.visitProperty(DotEnvPsiElementsVisitor.java:26)
	at ru.adelf.idea.dotenv.DotEnvPsiElementsVisitor.visitElement(DotEnvPsiElementsVisitor.java:19)
	at com.intellij.psi.impl.PsiElementBase.accept(PsiElementBase.java:274)
	at ru.adelf.idea.dotenv.psi.impl.DotEnvPropertyImpl.accept(DotEnvPropertyImpl.java:26)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.acceptChildren(SharedImplUtil.java:200)
	at com.intellij.psi.impl.source.PsiFileImpl.acceptChildren(PsiFileImpl.java:823)
	at ru.adelf.idea.dotenv.DotEnvVariablesProvider.getElements(DotEnvVariablesProvider.java:24)
	at ru.adelf.idea.dotenv.indexing.EnvironmentVariablesIndex.lambda$getIndexer$0(EnvironmentVariablesIndex.java:26)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapInput(MapReduceIndex.java:271)
	at com.intellij.util.indexing.VfsAwareMapReduceIndex.calculateUpdateData(VfsAwareMapReduceIndex.java:99)
	at com.intellij.util.indexing.impl.MapReduceIndex.update(MapReduceIndex.java:214)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:1645)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:1581)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.freezeFileTypeTemporarilyIn(FileTypeManagerImpl.java:511)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:1540)
	at com.intellij.util.indexing.FileBasedIndexImpl.indexFileContent(FileBasedIndexImpl.java:1528)
	at com.intellij.util.indexing.FileBasedIndexImpl.processRefreshedFile(FileBasedIndexImpl.java:1507)
	at com.intellij.util.indexing.FileBasedIndexImpl$VirtualFileUpdateTask.doProcess(FileBasedIndexImpl.java:1663)
	at com.intellij.util.indexing.FileBasedIndexImpl$VirtualFileUpdateTask.doProcess(FileBasedIndexImpl.java:1660)
	at com.intellij.util.indexing.UpdateTask.a(UpdateTask.java:76)
	at com.intellij.util.indexing.UpdateTask.processAll(UpdateTask.java:46)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:1682)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:725)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:691)
	at com.intellij.psi.stubs.StubIndexImpl.a(StubIndexImpl.java:346)
	at com.intellij.psi.stubs.StubIndexImpl.processElements(StubIndexImpl.java:323)
	at com.intellij.psi.stubs.StubIndex.getElements(StubIndex.java:145)
	at com.intellij.psi.stubs.StubIndex.getElements(StubIndex.java:134)
	at com.jetbrains.php.PhpIndexImpl.getFunctionsByName(PhpIndexImpl.java:290)
	at com.jetbrains.php.lang.psi.elements.impl.FunctionReferenceImpl.resolveGlobal(FunctionReferenceImpl.java:80)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:151)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:148)
	at com.intellij.psi.impl.source.resolve.ResolveCache.lambda$resolve$0(ResolveCache.java:145)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:144)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:175)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:167)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:143)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:129)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:389)
	at com.intellij.codeInsight.TargetElementUtil.b(TargetElementUtil.java:366)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:236)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:208)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.doCollectInformation(IdentifierHighlighterPass.java:92)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:437)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1127)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:430)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:429)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:229)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:403)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:170)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
``` 

**PHPStorm Info:**
PhpStorm 2017.3
Build #PS-173.3727.138, built on November 28, 2017
JRE: 1.8.0_144-b01 amd64
JVM: OpenJDK 64-Bit Server VM by Oracle Corporation
Linux 4.14.4-1-ARCH

Dont know it relevant or not. Im using Docker in my Project (with .env file) - it seems to do with that, but im not sure.

**Comment by adelf** [comment]
Seems, the reason described here: https://gist.github.com/androidfred/bc64da9e6a355b984d37439ed63ae16b

But I'll remove this dependency from next version. Thanks.

**Comment by adelf** [comment]
fixed in 0.6
